composer.json

```diff
@@ -37,7 +37,8 @@
         "drupal/sitestudio_config_management": "dev-develop",
         "drupal/sitestudio_gin": "^1.0",
         "drush/drush": "^10 || ^11 || ^12",
-        "mnsami/composer-custom-directory-installer": "^2.0"
+        "mnsami/composer-custom-directory-installer": "^2.0",
+        "nnnick/chartjs": "^4.4"
```

Comment:
Do we need this package in the require section of root composer.json?

Reply:
@chandan-singh7929 - i think we need to have 3.9 and 4.4 both, for the lower version to support

Reply:
Agree with @apathak18 this is not required as acquia_cms_common has this dependency.
cc @vishalkhode1 @chandan-singh7929